pkg/build/builder/sti.go

```diff
@@ -227,7 +227,7 @@ func (s *S2IBuilder) Build() error {
 	config.PullAuthentication, _ = dockercfg.NewHelper().GetDockerAuth(config.BuilderImage, dockercfg.PullAuthType)
 	config.IncrementalAuthentication, _ = dockercfg.NewHelper().GetDockerAuth(pushTag, dockercfg.PushAuthType)
 
-	glog.V(0).Infof("Creating a new S2I builder with build config: %#v\n", describe.DescribeConfig(config))
+	glog.V(4).Infof("Creating a new S2I builder with build config: %#v\n", describe.DescribeConfig(config))
```

Comment:
I understand the desire not to have this in the output, but this is the some of the most useful information we need when debugging user issues, so we're going to be asking everyone to recreate with a higher loglevel.

i suppose optimizing for failure isn't the right attitude, but i'm noting my objection anyway.